lfs/pointer.go

```diff
@@ -113,7 +113,7 @@ func DecodeFrom(reader io.Reader) ([]byte, *Pointer, error) {
 	written, err := reader.Read(buf)
 	output := buf[0:written]
 
-	if err != nil {
+	if err != nil && err != io.EOF {
```

Comment:
Is an `EOF` during pointer decoding really OK? That would be an error in the protocol, no?